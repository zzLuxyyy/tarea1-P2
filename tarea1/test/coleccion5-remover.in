crearColeccion

crearFecha 01/01/1952
crearLibro 121314151 "El viejo y el mar" "Ernest" "Hemingway" "Un viejo pescador enfrenta al mar en una lucha silenciosa" 1
agregarEnColeccion

crearFecha 01/01/1989
crearLibro 224512789 "El palacio de la luna" "Paul" "Auster" "Un joven cae en la pobreza y encuentra sentido en un viaje inesperado" 1
agregarEnColeccion

crearFecha 01/01/2007
crearLibro 234567891 "El informe de Brodeck" "Philippe" "Claudel" "Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra" 1
agregarEnColeccion

crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo" 3
agregarEnColeccion

crearFecha 01/09/1987
crearLibro 843397380 "Tokio Blues" "Haruki" "Murakami" "Un joven recuerda su vida universitaria marcada por la pérdida y el amor" 1
agregarEnColeccion

imprimirColeccion

removerDeColeccion 2

imprimirColeccion

removerDeColeccion 5

imprimirColeccion

removerDeColeccion 1

imprimirColeccion

liberarColeccion

Fin