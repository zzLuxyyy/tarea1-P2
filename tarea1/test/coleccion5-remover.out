1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 843397380
Título: Tokio Blues
Autor: Haruki Murakami
Descripción: Un joven recuerda su vida universitaria marcada por la pérdida y el amor
Género: 1
Fecha de edición: 1/9/1987
Libro 843760494
Título: Cien años de soledad
Autor: Gabriel García Márquez
Descripción: Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo
Género: 3
Fecha de edición: 5/6/1967
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
18>No se puede remover el libro con isbn 2 porque NO pertenece a la colección.
19>Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 843397380
Título: Tokio Blues
Autor: Haruki Murakami
Descripción: Un joven recuerda su vida universitaria marcada por la pérdida y el amor
Género: 1
Fecha de edición: 1/9/1987
Libro 843760494
Título: Cien años de soledad
Autor: Gabriel García Márquez
Descripción: Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo
Género: 3
Fecha de edición: 5/6/1967
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
20>No se puede remover el libro con isbn 5 porque NO pertenece a la colección.
21>Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 843397380
Título: Tokio Blues
Autor: Haruki Murakami
Descripción: Un joven recuerda su vida universitaria marcada por la pérdida y el amor
Género: 1
Fecha de edición: 1/9/1987
Libro 843760494
Título: Cien años de soledad
Autor: Gabriel García Márquez
Descripción: Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo
Género: 3
Fecha de edición: 5/6/1967
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
22>No se puede remover el libro con isbn 1 porque NO pertenece a la colección.
23>Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 843397380
Título: Tokio Blues
Autor: Haruki Murakami
Descripción: Un joven recuerda su vida universitaria marcada por la pérdida y el amor
Género: 1
Fecha de edición: 1/9/1987
Libro 843760494
Título: Cien años de soledad
Autor: Gabriel García Márquez
Descripción: Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo
Género: 3
Fecha de edición: 5/6/1967
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
24>25>Fin.
